daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1438,8 +1438,6 @@ private[lf] object SBuiltin {
 
   /** $insertFetch[tid]
     *    :: ContractId a
-    *    -> List Party    (signatories)
-    *    -> List Party    (observers)
     *    -> Optional {key: key, maintainers: List Party}  (template key, if present)
     *    -> ()
```

Comment:
Don't we also now return the templateArg?

Reply:
Yes. Thanks. Fixed.

Reply:
I would be very happy to strip out all these worthless comments.
In my opinion they add nothing to the code, which is the real documentation!

Note. This comment was wrong before I started.